plugins/example/ahda/client.py

```
@@ -0,0 +1,143 @@
+from fastapi import APIRouter, Request, HTTPException, Form, Query, BackgroundTasks, Body
+from fastapi.responses import HTMLResponse, FileResponse, JSONResponse
```

Ruff

`fastapi.responses.JSONResponse` imported but unused

Remove unused import: `fastapi.responses.JSONResponse`

(F401)